Cython/Includes/libcpp/string.pxd

```diff
@@ -5,7 +5,7 @@ from libc.string cimport const_char
 
 cdef extern from "<string>" namespace "std" nogil:
 
-    size_t npos = -1
+    const size_t npos "std::string::npos"
```

Comment:
Why did you chose this spelling over the separate `cdef extern` block given in the [ticket](https://github.com/cython/cython/issues/4268#issue-937625219)?

Reply:
Because it's shorter and seems to generate equivalent code.

Reply:
I would prefer if Cython knew from the declaration that the name uses a different C++ namespace, 1) because why not and 2) because I can't tell in what way C++ namespace handling will change in the future in Cython, so better declare it correctly right now.

Explicit cnames are more of a last-resort way to get what your C(++) compiler wants. We should instead try to give good examples in the declarations that we ship. Users may end up actually reading them as The Right Way To Do It.

Reply:
Alright, I changed it. I would even prefer if I could declare it as a static member of `std::string`, because that's what it is. But then Cython would translate it as `std::string.npos`, which obviously doesn't work.